logpolicy/logpolicy.go

```diff
@@ -374,6 +381,7 @@ func New(collection string) *Policy {
 	}
 
 	c := logtail.Config{
+		BaseURL:    fmt.Sprintf("https://%s", newc.Target),
```

Comment:
oh, it's a URL suffix!

let's just make it a URL then, so people can use http for local development?